content/ri/release-notes/v1.13.0.md

```diff
@@ -6,6 +6,18 @@ description: RedisInsight v1.13.0
 weight: 7
 ---
 
+## 1.13.2 (May 2023)
+
+This is the maintenance release of RedisInsight 1.13 (v1.13.2)!
```

Comment:
```suggestion
This is the maintenance release of RedisInsight 1.13 (v1.13.2).
```

Reply:
RedisInsight version 1.X has been retired on April 30, 2023, and will no longer be supported.
To continue using the best RedisInsight features and capabilities, download the latest RedisInsight version 2.Y from our [website](https://redis.com/redis-enterprise/redis-insight/) or install it from any app store.

This is the maintenance release of RedisInsight 1.14 (v1.14.0).